init.lua

```diff
@@ -15,6 +15,8 @@ if mineclone_path then -- means MineClone 2 is loaded
 	moditems.boxart = "bgcolor[#d0d0d0;false]listcolors[#9d9d9d;#9d9d9d;#5c5c5c;#000000;#ffffff]"
 	moditems.trashbin_groups = {pickaxey=1,axey=1,handy=1,swordy=1,flammable=1,destroy_by_lava_flow=1,craftitem=1}
 	moditems.dumpster_groups = {pickaxey=1,axey=1,handy=1,swordy=1,flammable=0,destroy_by_lava_flow=0,craftitem=1}
+	moditems.inventory_row_size = 9
```

Comment:
Did you mean *columns*? Because you're using this variable for the `x` direction.

Reply:
It's the number of element in a row. I can change to use the name `slot_per_row`. Is it better ?

Reply:
Well yes. That's also the column count but worded differently. That's OK too.